src/app/portlets/dot-edit-content/shared/iframe-edit-mode.css.ts

```diff
@@ -93,4 +89,127 @@ export const EDIT_PAGE_CSS = `
             transform: rotate(360deg);
         }
     }
+    
+    .dotedit-container__toolbar {
+        position:relative;
+        margin:0 0 -26px 8px;
+        display: table;
+    }
+    
+    .dotedit-container__toolbar button,
+    .dotedit-contentlet__toolbar button {
+        width: 32px;
+        height: 32px;
+        border: none;
+        border-radius: 16px;
+        box-shadow: 0 1px 3px rgba(0, 0, 0, 0.12), 0 1px 2px rgba(0, 0, 0, 0.24);
```

Comment:
Create vars for the shadows too, we did that in the scss, we can do it here too.